capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -104,7 +104,8 @@ class AdminClubPage extends Component {
               key={r.id}
               user={r}
               club={this.state.club}
-              fetchData={this.fetchData} />
+              fetchData={this.fetchData}
+              updateFriendsList={this.props.updateFriendsList} />
```

Comment:
nit: Move `/>` to the next line so this doesn't get caught in the diff if you add more props.

Reply:
That's a good point! Thanks